Common/Orders/Fills/FillModel.cs

```diff
@@ -31,41 +30,23 @@ public class FillModel : IFillModel
         /// <param name="asset">Security asset we're filling</param>
         /// <param name="order">Order packet to model</param>
         /// <returns>Order fill information detailing the average price and quantity filled.</returns>
-        /// <seealso cref="SecurityTransactionModel.StopMarketFill"/>
-        /// <seealso cref="SecurityTransactionModel.LimitFill"/>
+        [Obsolete("This was left for retro compatibility, see new MarketFill(FillModelContext context)")]
```

Comment:
I think we can leave these methods non-obsolete. We're only removing them from the `IFillModel` definition. No need to remove them from `FillModel`. It basically _moves_ the switch statement from `BacktestingBrokerage` into `FillModel.Fill`